000-vip-init.php

```diff
@@ -27,7 +27,7 @@
 	define( 'VIP_GO_ENV', false );
 }
 
-// For backwards compatibility - always true.
+// For backwards compatibility - always true on a VIP Go environment.
```

Comment:
The constant is pretty heavily used so I think we should probably remove the part about back compat.